.github/workflows/CI.yml

```diff
@@ -69,6 +72,9 @@ jobs:
           profile: minimal
           toolchain: ${{ matrix.toolchain }}
           override: true
+          # x86_64 testing blocked on https://github.com/actions-rs/toolchain/issues/16
+          target: aarch64-apple-ios
```

Comment:
Won't that make it the target for every tests?

Reply:
This will only download the extra target, but won't use it unless its in the matrix. The time it takes to download appears negligible and within margin of error. [This past PR CI job](https://github.com/GuillaumeGomez/sysinfo/pull/399/checks?check_run_id=1683614711) took 14 seconds on the `Install Toolchain` step but it only took 11 seconds on this PR.

Reply:
Could you add a comment with this explanation please?

Reply:
Added.